app/dashboard/notifications.py

```diff
@@ -223,3 +224,97 @@ def maybe_market_to_email(b, event_name, txid):
 
     return len(to_emails)
 
+def maybe_post_on_craigslist(bounty):
+    CRAIGSLIST_URL = 'https://www.craigslist.org'
+    MAX_URLS = 10
+
+    import mechanicalsoup
+    from random import randint
+    from app.utils import fetch_last_email_id,fetch_mails_since_id
+
+    browser = mechanicalsoup.StatefulBrowser()
+    browser.open(CRAIGSLIST_URL) # open craigslist
```

Comment:
what locale does `CRAIGSLIST_URL` point to?  i see in the email inbox that these are being posted to bangalore?

Reply:
is it possible to enumerate several different locales?

Reply:
We are hitting 'www.craigslist.com' which gets redirects by craigslist. It will get redirected to the locale where our server is hosted. We can enumerate multiple locales as well.

Reply:
yes please!